packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceSourceDelete.test.ts

```diff
@@ -27,7 +27,7 @@ describe('Force Source Delete', () => {
     const sourcePath = path.join('example', 'path');
     const sourceDeleteCommand = executor.build({ filePath: sourcePath });
     expect(sourceDeleteCommand.toCommand()).to.equal(
-      `sfdx force:source:delete --sourcepath ${sourcePath} --noprompt`
+      `sfdx force:source:delete --sourcepath ${sourcePath} --noprompt --tracksource`
```

Comment:
👍🏽

Reply:
The changes in the PR are looking great!  What do you think about writing a new jest test to replace this test?